db/seeds/npq_applications.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "securerandom"
+
+module Seeds
+  class NPQApplication
+    attr_reader :cpd_lead_provider
+
+    def initialize(cpd_lead_provider:)
+      @cpd_lead_provider = cpd_lead_provider
+    end
+
+    def call
+      NPQ::BuildApplication.call(
```

Comment:
```suggestion
      return if Rails.env.production?
      
      NPQ::BuildApplication.call(
```

What do you think?

Reply:
sure. it's not loaded so, one has to be explicitly require it for it to be available